_posts/2015-03-16-CocoaPods.org-Take-Two.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "CocoaPods.org Take Two"
+author: orta
+categories: cocoapods.org releases website
+---
+
+TL;DR: [CocoaPods.org][1] version 2 has been released, with support for inline deep search results, and pod pages.
+
+<!-- more -->
+
+
+## This took way longer than expected.
+
+The CocoaPods web site is an interesting beast. In October 2013 we set around [rewriting][2] the CocoaPods web pages. We centralized the documentation, created a blog and redesigned everything. 
+
+We settled on using a mix of middleman pages and jekyll. Given that, at the time, there was no central database for CocoaPods  it made sense to try and keep it as a static site for speed and maintainability.
```

Comment:
Links to `middleman` and `jekyll` websites, for those who don't know what those are.